### PR TITLE
py-emodel-generalisation: bump to 0.2.5 and add to whitelist

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

File: bluebrain/deployment/environments/applications_science.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ spack:
           - py-efel
           - py-synthesis-workflow
           - py-visual-cortex-analysis
+          - py-emodel-generalisation
```
Comment (matz-e) [inline bluebrain/deployment/environments/applications_science.yaml]: This only enables a module, but does not build the software. Please add the same entry after line 45 :)

Comment (arnaudon) [reply bluebrain/deployment/environments/applications_science.yaml]: ah right, thanks!

**Comment by arnaudon** [comment]
I'm guessing the llvm error here is not my fault, right?

**Comment by arnaudon** [comment]
thanks for tyring to merge this! I'm just back here, I could have done it! Hopefully this will do!

**Comment by matz-e** [comment]
> thanks for tyring to merge this! I'm just back here, I could have done it! Hopefully this will do!

no prob. I had to fix the LLVM issue separately, rebased the inital "fix" commits away... hopefully it'll build now and all the tests work :crossed_fingers:

**Review by matz-e — DISMISSED** [review]

**Comment by matz-e** [comment]
@arnaudon `git push --force-with-lease`, please :)

Also, while you're updating stuff, is there a hard requirement for this:
```
    "ipyparallel>=6.3,<7",
```
?

That is the source of all this pain :)

**Review by matz-e — APPROVED** [review]
Please test module (I relaxed the `ipyparallel` requirement - 6.X is *four* years old)

**Comment by arnaudon** [comment]
Ah sorry, for some reason, I don't get notifications on spack pr...
I will relax ipyparallel in the next release, but I can't remember why we had this (@adrien-berchet  may know). 
I didn't know the force-with-lease, I'll use it!

**Comment by matz-e** [comment]
Thank you!

**Comment by adrien-berchet** [comment]
`ipyparallel` was relaxed in `bluepyparallel-v0.2.0` so I guess you can import these changes in `emodel-generalisation`

**Pull request merged**